src/pagination/pagination.component.ts

```diff
@@ -158,6 +165,18 @@ export class Pagination {
 	 * @type {Model}
 	 */
 	@Input() model: PaginationModel;
+	/**
+ 	 * Set to `true` to disable the backward/forward buttons.
+	 */
+	@Input() disabled = false;
+	/**
+	 * Set to `true` to disable the select box that changes the page.
+	 */
+	@Input() pageInputDisabled = false;
```

Comment:
I'm not sure I understand when this would be used ... `disabled` should disable everything ... this removes the select box, but also disabled the buttons. 

Is there a spec or an issue for this particular change?

Reply:
In [Carbon design systems](https://www.carbondesignsystem.com/components/pagination/code) the `disabled` feature just disabled the buttons. So I checked [react](http://react.carbondesignsystem.com/?selectedKind=Pagination&selectedStory=Pagination&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) and that was the way they disabled their select

Reply:
Right, but `pageInputDisabled` there just removes the select, it doesn't remove the select _and_ disable the buttons.